docs/sources/datasources/google-cloud-monitoring/_index.md

```diff
@@ -14,13 +14,12 @@ Grafana ships with built-in support for Google Cloud Monitoring. Add it as a dat
 
 ## Google Cloud Monitoring settings
 
-To access Google Cloud Monitoring settings, hover your mouse over the **Configuration** (gear) icon, then click **Data Sources**, and then click the Google Cloud Monitoring data source.
+To access Google Cloud Monitoring settings, hover your mouse over the **Configuration** (gear) icon, then click **Data Sources**, click **Add data source**, then click the Google Cloud Monitoring data source.
```

Comment:
Let's keep the "and", as it is needed here.

Reply:
Done